src/client-rs/src/client/cluster.rs

```diff
@@ -14,16 +14,18 @@ use super::*;
 use flv_future_aio::net::tls::AllDomainConnector;
 use std::convert::TryFrom;
 
-/// Client connection to cluster
-pub struct ClusterClient {
+/// socket to Cluster
+pub struct ClusterSocket {
```

Comment:
I like this name 👍

Reply:
Yes, that's purpose of new name.  So basically, we have 2 abstraction:
- Socket (transport layer)
- Client (API).